reports/generate_reports_data.py

```diff
@@ -133,50 +133,52 @@ def generate_daily_service_hours(itp_id: int, date_start, date_end):
 
 
 @cache
-def _file_check():
+def _guideline_check():
     return (
         LazyTbl(
             engine,
-            "mart_gtfs_quality.fct_monthly_reports_site_organization_file_checks",
+            "mart_gtfs_quality.fct_monthly_reports_site_organization_guideline_checks",
         )
         >> select(
             _.organization_itp_id,
             _.publish_date,
             _.date_checked,
-            _.reason,
-            _.filename,
-            _.file_present,
+            _.feature,
+            _.check,
+            _.status,
         )
         >> collect()
     )
 
 
-def generate_file_check(itp_id: int, publish_date):
-    importance = ["Visual display", "Navigation", "Fares", "Technical contacts"]
+def generate_guideline_check(itp_id: int, publish_date):
+    importance = [
```

Comment:
If we want to go really galaxy brain, we could think about whether there are ways that we want to show something in the UI for places where the data has changed since the prior month

Reply:
let's spin the "UI highlights change" idea into a separate issue to be addressed later

Reply:
https://github.com/cal-itp/reports/issues/257